google-cloud-spanner/lib/google/cloud/spanner/service.rb

```diff
@@ -275,19 +277,23 @@ def create_session database_name, labels: nil
           session = Google::Spanner::V1::Session.new labels: labels if labels
           execute do
             service.create_session database_name, session: session,
-                                                  options: opts
+                                                 options: opts
           end
         end
 
-        def batch_create_sessions database_name, session_count, labels: nil
+        def batch_create_sessions \
```

Comment:
Is there any style guide for this change? Otherwise, I would think it is not necessary, including all changes below in this file.

Reply:
Using rubocop we enforce style. These changes due to previously I have added endpoint_uri in all client methods. I have reverted to old syntax  for all client method.